test/test-helpers/common.js

```diff
@@ -307,13 +328,88 @@ export default function (repository, dbClient) {
                         type: 'car',
                         color: 'black',
                         pet_id: 2
+                    }),
+                    repository.Models.MovieModel.forge().save({
+                        id: 1,
+                        name: 'Gone',
+                        type: 'null'
+                    }),
+                    repository.Models.MovieModel.forge().save({
+                        id: 2,
+                        name: 'Spider',
+                        type: null
                     })
                 );
             }).then(() => done());
         });
 
         describe('passing no parameters', () => {
 
+            it('should return string null while passing string', (done) => {
+
+                repository.Models.MovieModel
+                    .forge()
+                    .fetchJsonApi({
+                        filter: {
+                            'type': 'null'
+                        }
+                    })
+                    .then((result) => {
+
+                        expect(result.models).to.have.length(1);
+                        expect(result.models[0].get('name')).to.equal('Gone');
+                        done();
+                    });
+            });
+
+            it('should return value null while passing value', (done) => {
+
+                repository.Models.MovieModel
+                    .forge()
+                    .fetchJsonApi({
+                        filter: {
+                            'type': 'null'
+                        }
+                    })
+                    .then((result) => {
+
+                        expect(result.models).to.have.length(1);
+                        expect(result.models[0].get('name')).to.equal('Gone');
+                        done();
+                    });
+            });
+
+            before((done) => {
+
+                repository.plugin(JsonApiParams, {
+                    nullString: '_null_'
+                });
+                done();
+            });
+
+            it('should return value null while passing _null_', (done) => {
+
+                repository.Models.MovieModel
+                    .forge()
+                    .fetchJsonApi({
+                        filter: {
+                            'type': '_null_'
+                        }
+                    })
+                    .then((result) => {
+
+                        expect(result.models).to.have.length(1);
```

Comment:
To make this test more thorough, another piece of data should have `_null_` as the type value and this should still only return one record